diff --git a/dv/env/pwm_env_pkg.sv b/dv/env/pwm_env_pkg.sv
index 6121d17..f48d5f7 100644
--- a/dv/env/pwm_env_pkg.sv
+++ b/dv/env/pwm_env_pkg.sv
@@ -21,8 +21,8 @@ package pwm_env_pkg;
   `include "dv_macros.svh"
 
   // The DUT has a single alert signal.
-  parameter uint NUM_ALERTS = 1;
-  parameter string LIST_OF_ALERTS[] = {"fatal_fault"};
+  parameter uint NUM_ALERTS = 0;
+  parameter string LIST_OF_ALERTS[] = {};
 
   // Constants.
   parameter bit [31:0] MAX_32 = 32'hFFFF_FFFF;
diff --git a/dv/pwm_sim_cfg.hjson b/dv/pwm_sim_cfg.hjson
index 44503dc..efd204e 100644
--- a/dv/pwm_sim_cfg.hjson
+++ b/dv/pwm_sim_cfg.hjson
@@ -18,33 +18,33 @@
   fusesoc_core: lowrisc:dv:pwm_sim:0.1
 
   // Testplan hjson file.
   testplan: "{proj_root}/hw/ip/pwm/data/pwm_testplan.hjson"
 
   // RAL spec - used to generate the RAL model.
-  ral_spec: "{proj_root}/hw/ip/pwm/data/pwm.hjson"
+  ral_spec: "{proj_root}/hw/vendor/lowrisc_ip_main/ip/pwm/data/pwm.hjson"
 
   // Import additional common sim cfg files.
   import_cfgs: [// Project wide common sim cfg file
-                "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/common_sim_cfg.hjson",
                 // Common CIP test lists
-                "{proj_root}/hw/dv/tools/dvsim/tests/csr_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/alert_test.hjson",
-                //"{proj_root}/hw/dv/tools/dvsim/tests/stress_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/sec_cm_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/tl_access_tests.hjson"]
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/csr_tests.hjson",
+                //"{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/alert_test.hjson",
+                //"{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/stress_tests.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/sec_cm_tests.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/tl_access_tests.hjson"]
 
   // Add additional tops for simulation.
   sim_tops: ["pwm_bind", "sec_cm_prim_onehot_check_bind"]
 
   // Coverage exclusion
-  xcelium_cov_refine_files: ["{proj_root}/hw/ip/pwm/dv/cov/pwm_unr_excl.vRefine"]
+  xcelium_cov_refine_files: ["{proj_root}/hw/vendor/lowrisc_ip_main/ip/pwm/dv/cov/pwm_unr_excl.vRefine"]
 
   overrides: [
     {
       // Override the base ccf coverage configuration file for the "default" build mode (used for tests
       // other than the CSR tests). This will include the default cover.ccf file as its first item.
       name: default_xcelium_cov_cfg_file
-      value: "{proj_root}/hw/ip/pwm/dv/cov/cover.ccf"
+      value: "{proj_root}/hw/vendor/lowrisc_ip_main/ip/pwm/dv/cov/cover.ccf"
     }
   ]
 
diff --git a/dv/tb.sv b/dv/tb.sv
index 5f007e8..d6e2e42 100644
--- a/dv/tb.sv
+++ b/dv/tb.sv
@@ -31,6 +31,8 @@ module tb;
 
   `DV_ALERT_IF_CONNECT()
 
+  assign alert_tx = '0;
+
   // dut
   pwm dut (
     .clk_i         (clk),
@@ -42,9 +44,6 @@ module tb;
     .tl_i          (tl_if.h2d),
     .tl_o          (tl_if.d2h),
 
-    .alert_rx_i    (alert_rx),
-    .alert_tx_o    (alert_tx),
-
     .cio_pwm_o     (cio_pwm),
     .cio_pwm_en_o  (cio_pwm_en)
   );
diff --git a/dv/cov/cover.ccf b/dv/cov/cover.ccf
index 5b8de80..86ef5c2 100644
--- a/dv/cov/cover.ccf
+++ b/dv/cov/cover.ccf
@@ -6,4 +6,4 @@
 include_ccf ${dv_root}/tools/xcelium/cover.ccf
 
 // Exclude some impossible toggle coverage
-set_toggle_excludefile -bitexclude ${proj_root}/hw/ip/pwm/dv/cov/toggle.exclude
+set_toggle_excludefile -bitexclude ${proj_root}/hw/vendor/lowrisc_ip_main/ip/pwm/dv/cov/toggle.exclude
