diff --git a/dv/env/i2c_env_pkg.sv b/dv/env/i2c_env_pkg.sv
index 56323d0..07673e5 100644
--- a/dv/env/i2c_env_pkg.sv
+++ b/dv/env/i2c_env_pkg.sv
@@ -81,8 +81,8 @@ package i2c_env_pkg;
   parameter uint I2C_ACQ_FIFO_DEPTH = i2c_reg_pkg::AcqFifoDepth;
 
   // alerts
-  parameter uint NUM_ALERTS = i2c_reg_pkg::NumAlerts;
-  parameter string LIST_OF_ALERTS[] = {"fatal_fault"};
+  parameter uint NUM_ALERTS = 0;
+  parameter string LIST_OF_ALERTS[] = {};
 
   function automatic i2c_acqdata_item acq2item(bit [bus_params_pkg::BUS_DW-1:0] data);
     i2c_acqdata_item item;
diff --git a/dv/i2c_sim_cfg.hjson b/dv/i2c_sim_cfg.hjson
index 4edfc9c..bbeb8b0 100644
--- a/dv/i2c_sim_cfg.hjson
+++ b/dv/i2c_sim_cfg.hjson
@@ -18,26 +18,26 @@
   fusesoc_core: lowrisc:dv:i2c_sim:0.1
 
   // Testplan hjson file.
-  testplan: "{proj_root}/hw/ip/i2c/data/i2c_testplan.hjson"
+  testplan: "{proj_root}/hw/vendor/lowrisc_ip/ip/i2c/data/i2c_testplan.hjson"
 
   // RAL spec - used to generate the RAL model.
-  ral_spec: "{proj_root}/hw/ip/i2c/data/i2c.hjson"
+  ral_spec: "{proj_root}/hw/vendor/lowrisc_ip/ip/i2c/data/i2c.hjson"
 
   // Import additional common sim cfg files.
   import_cfgs: [// Project wide common sim cfg file
-                "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/common_sim_cfg.hjson",
                 // Common CIP test lists
-                "{proj_root}/hw/dv/tools/dvsim/tests/csr_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/alert_test.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/intr_test.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/sec_cm_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/tl_access_tests.hjson"]
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/csr_tests.hjson",
+                //"{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/alert_test.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/intr_test.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/sec_cm_tests.hjson",
+                "{proj_root}/hw/vendor/lowrisc_ip/dv/tools/dvsim/tests/tl_access_tests.hjson"]
 
   // Add additional tops for simulation.
   sim_tops: ["i2c_bind", "sec_cm_prim_onehot_check_bind"]
 
   // Add coverage exclusion file
-  vcs_cov_excl_files: ["{proj_root}/hw/ip/i2c/dv/cov/i2c_cov_excl.el"]
+  vcs_cov_excl_files: ["{proj_root}/hw/vendor/lowrisc_ip/ip/i2c/dv/cov/i2c_cov_excl.el"]
 
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
@@ -53,7 +53,7 @@
   overrides: [
     {
       name: default_vcs_cov_cfg_file
-      value: "-cm_hier {proj_root}/hw/dv/tools/vcs/cover.cfg+{proj_root}/hw/dv/tools/vcs/common_cov_excl.cfg+{proj_root}/hw/ip/i2c/dv/cov/cov_excl.cfg"
+      value: "-cm_hier {proj_root}/hw/vendor/lowrisc_ip/dv/tools/vcs/cover.cfg+{proj_root}/hw/vendor/lowrisc_ip/dv/tools/vcs/common_cov_excl.cfg+{proj_root}/hw/vendor/lowrisc_ip/ip/i2c/dv/cov/cov_excl.cfg"
     }
   ]
 
diff --git a/dv/tb/tb.sv b/dv/tb/tb.sv
index 5ae4816..f70036f 100644
--- a/dv/tb/tb.sv
+++ b/dv/tb/tb.sv
@@ -77,6 +77,8 @@ module tb;
   // clk and rst_n is used for alert_if in `DV_ALERT_IF_CONNECT
   `DV_ALERT_IF_CONNECT()
 
+  assign alert_tx = '0;
+
   // dut
   i2c dut (
     .clk_i                   (clk        ),
@@ -86,9 +88,6 @@ module tb;
     .tl_i                    (tl_if.h2d  ),
     .tl_o                    (tl_if.d2h  ),
 
-    .alert_rx_i              (alert_rx   ),
-    .alert_tx_o              (alert_tx   ),
-
     .cio_scl_i               (cio_scl               ),
     .cio_scl_o               (/*hardcoded to 0*/    ),
     .cio_scl_en_o            (cio_scl_en            ),
diff --git a/data/i2c_testplan.hjson b/i2c/data/i2c_testplan.hjson
index 07120ff..f194499 100644
--- a/data/i2c_testplan.hjson
+++ b/data/i2c_testplan.hjson
@@ -3,10 +3,8 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: "i2c"
-  import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
-                     "i2c_sec_cm_testplan.hjson"]
+  import_testplans: ["hw/vendor/lowrisc_ip/dv/tools/dvsim/testplans/csr_testplan.hjson",
+                     "hw/vendor/lowrisc_ip/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
+                     "hw/vendor/lowrisc_ip/dv/tools/dvsim/testplans/tl_device_access_types_wo_intg_testplan.hjson"]
   testpoints: [
     //-----------------------------------------------
