From 496c8eed91878b92455cba587484720e13919b98 Mon Sep 17 00:00:00 2001
From: Samuel Riedel <sriedel@lowrisc.org>
Date: Tue, 1 Apr 2025 08:29:37 +0000
Subject: [PATCH 1/2] Fix `scisemi_ip_riscv_dbg.core` file for xcelium and vcs

---
 scisemi_ip_riscv_dbg.core | 17 ++---------------
 1 file changed, 2 insertions(+), 15 deletions(-)

diff --git a/scisemi_ip_riscv_dbg.core b/scisemi_ip_riscv_dbg.core
index 138e911..becc53b 100644
--- a/scisemi_ip_riscv_dbg.core
+++ b/scisemi_ip_riscv_dbg.core
@@ -19,24 +19,11 @@ filesets:
       - src/dm_mem.sv
       - src/dmi_cdc.sv
       - src/dmi_jtag.sv
-    file_type: systemVerilogSource
-
-  files_vivado:
-    files:
-      - src/dmi_bscane_tap.sv
-    file_type: systemVerilogSource
-
-  files_verilator:
-    depend:
-      # common waivers
-      - lowrisc:lint:common
-    files:
-      - src/dmi_jtag_tap.sv
+      - "!tool_vivado ? (src/dmi_jtag_tap.sv)"
+      - "tool_vivado ? (src/dmi_bscane_tap.sv)"
     file_type: systemVerilogSource
 
 targets:
   default:
     filesets:
       - files_src
-      - tool_verilator ? (files_verilator)
-      - tool_vivado ? (files_vivado)
-- 
2.34.1

