diff --git a/autogen_banner.py b/autogen_banner.py
index 00a104e..5aed6a6 100644
--- a/autogen_banner.py
+++ b/autogen_banner.py
@@ -5,7 +5,7 @@
 from textwrap import fill
 
 LICENSE_BANNER = (
+    "Copyright lowRISC contributors (Sunburst project).\n"
-    "Copyright lowRISC contributors (OpenTitan project).\n"
     "Licensed under the Apache License, Version 2.0, see LICENSE for details.\n"
     "SPDX-License-Identifier: Apache-2.0")
 
