diff --git a/rtl/aon_timer.sv b/rtl/aon_timer.sv
index 066f8e8b6e..ad25a3630f 100644
--- a/rtl/aon_timer.sv
+++ b/rtl/aon_timer.sv
@@ -6,9 +6,7 @@
 `include "prim_assert.sv"
 
 module aon_timer import aon_timer_reg_pkg::*;
-#(
-  parameter logic [NumAlerts-1:0] AlertAsyncOn = {NumAlerts{1'b1}}
-) (
+(
   input  logic                clk_i,
   input  logic                clk_aon_i,
   input  logic                rst_ni,
@@ -18,10 +16,6 @@ module aon_timer import aon_timer_reg_pkg::*;
   input  tlul_pkg::tl_h2d_t   tl_i,
   output tlul_pkg::tl_d2h_t   tl_o,
 
-  // Alerts
-  input  prim_alert_pkg::alert_rx_t [NumAlerts-1:0] alert_rx_i,
-  output prim_alert_pkg::alert_tx_t [NumAlerts-1:0] alert_tx_o,
-
   // clk_i domain
   input  lc_ctrl_pkg::lc_tx_t lc_escalate_en_i,
   output logic                intr_wkup_timer_expired_o,
@@ -61,8 +55,6 @@ module aon_timer import aon_timer_reg_pkg::*;
   // Reset signals
   logic                      aon_rst_req_set;
   logic                      aon_rst_req_d, aon_rst_req_q;
-  // Alert signals
-  logic [NumAlerts-1:0]      alert_test, alerts;
 
   //////////////////////////////
   // Register Write Interface //
@@ -98,35 +90,9 @@ module aon_timer import aon_timer_reg_pkg::*;
     .reg2hw,
     .hw2reg,
 
-    // SEC_CM: BUS.INTEGRITY
-    .intg_err_o (alerts[0])
+    .intg_err_o (0)
   );
 
-  ////////////
-  // Alerts //
-  ////////////
-
-  assign alert_test = {
-    reg2hw.alert_test.q &
-    reg2hw.alert_test.qe
-  };
-
-  for (genvar i = 0; i < NumAlerts; i++) begin : gen_alert_tx
-    prim_alert_sender #(
-      .AsyncOn(AlertAsyncOn[i]),
-      .IsFatal(1'b1)
-    ) u_prim_alert_sender (
-      .clk_i,
-      .rst_ni,
-      .alert_test_i  ( alert_test[i] ),
-      .alert_req_i   ( alerts[0]     ),
-      .alert_ack_o   (               ),
-      .alert_state_o (               ),
-      .alert_rx_i    ( alert_rx_i[i] ),
-      .alert_tx_o    ( alert_tx_o[i] )
-    );
-  end
-
   // Lifecycle sync
   prim_lc_sync #(
     .NumCopies(3)
@@ -257,13 +223,9 @@ module aon_timer import aon_timer_reg_pkg::*;
   // clk_i domain
   `ASSERT_KNOWN(TlODValidKnown_A, tl_o.d_valid)
   `ASSERT_KNOWN(TlOAReadyKnown_A, tl_o.a_ready)
-  `ASSERT_KNOWN(AlertsKnown_A, alert_tx_o)
   `ASSERT_KNOWN(IntrWkupKnown_A, intr_wkup_timer_expired_o)
   `ASSERT_KNOWN(IntrWdogKnown_A, intr_wdog_timer_bark_o)
   // clk_aon_i domain
   `ASSERT_KNOWN(WkupReqKnown_A, wkup_req_o, clk_aon_i, !rst_aon_ni)
   `ASSERT_KNOWN(RstReqKnown_A, aon_timer_rst_req_o, clk_aon_i, !rst_aon_ni)
-
-  // Alert assertions for reg_we onehot check
-  `ASSERT_PRIM_REG_WE_ONEHOT_ERROR_TRIGGER_ALERT(RegWeOnehotCheck_A, u_reg, alert_tx_o[0])
 endmodule
